### DISCUSSION TITLE
No build tool message

### DISCUSSION THREAD

**Discussion by ByteDrummer** [scalameta/coc-metals#457] [Q&A]
Just installed this plugin and I'm getting the following message when I open a Scala file: `[coc.nvim] No build tool`. I have sbt installed so I'm not sure what it means by this.

**Comment by ckipp01** [comment]
Hey @ByteDrummer when you open a file it will try to find your `build.sbt` file and that will be treated as the root of your workspace. Are you just opening a single Scala file or are you opening a valid sbt build?
